test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -260,28 +260,29 @@
 (deftest error-handling-test
   (with-status-service app
     [foo-service]
-    (testing "returns a 400 when an invalid level is queried for"
-      (let [resp (http-client/get "http://localhost:8180/status/v1/services?level=bar")]
-        (is (= 400 (:status resp)))
-        (is (= {"error" {"type" "request-data-invalid"
-                         "message" "Invalid level: :bar"}}
-              (parse-response resp)))))
-    (testing "returns a 400 when a non-integer status-version is queried for"
-      (let [resp (http-client/get (str "http://localhost:8180/status/v1/"
-                                    "services/foo?service_status_version=abc"))]
-        (is (= 400 (:status resp)))
-        (is (= {"error" {"type"    "request-data-invalid"
-                         "message" (str "Invalid service_status_version. "
-                                     "Should be an integer but was abc")}}
-              (parse-response resp)))))
-    (testing "returns a 400 when a non-existent status-version is queried for"
-      (let [resp (http-client/get (str "http://localhost:8180/status/v1/"
-                                    "services/foo?service_status_version=3"))]
-        (is (= 400 (:status resp)))
-        (is (= {"error" {"type"    "service-status-version-not-found"
-                         "message" (str "No status function with version 3 "
-                                     "found for service foo")}}
-              (parse-response resp)))))))
+    (with-test-logging
```

Comment:
These tests create log messages that clutter up the test output without this test.

That was missed when we moved to use the ring-middleware wrapper previously. While updating these tests I figured it would be good to clean up the log output. I can pull it out, or move it into it's own commit. I should have at least put it into its own commit.